Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN echo "Building for $GOARCH" \
     && GOARCH=${GOARCH} hack/build.sh 
 
 # Final stage: the running container.
-FROM scratch AS final
+FROM gcr.io/distroless/static:nonroot AS final
 
 # Import the Certificate-Authority certificates for enabling HTTPS.
 COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
```

Comment:
ca-certificates are included in distroless already, no need to copy from builder